README.md

```diff
@@ -0,0 +1,63 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] It seems your node_modules folder is currently on this branch, kindly delete and add it to the gitignore file before merging to the `main` branch.